pages/product.js

```diff
@@ -0,0 +1,39 @@
+const { I } = inject();
+
+module.exports = {
+  priceText: { xpath: '//div[@class="price"]/span' },
+  dropdownElement: { xpath: "//a[contains(@id, 'sbSelector')]" },
+  deliveryPriceText: { xpath: "//strong[text()='Flat Shipping Rate:']/ancestor::tr/td[2]" },
+  totalPriceText: { xpath: "//strong[text()='Total:']/ancestor::tr/td[2]" },
+  order: { xpath: '//*[@id="content"]/div[1]/table/tbody/tr[1]/td[1]' },
+
+
+  async getProductPrice() {
+    return (await I.grabTextFrom(this.priceText)).replace(/[^0-9\.]+/g, "");
```

Comment:
move `replace()` to steps_file.js as separate method and use in all cases, where you need cleanup